pkg/target/host.go

```diff
@@ -189,10 +192,14 @@ func (h *Host) Connect(updateHostHealthStatus chan *HostStatus, attemptCount int
 	return true
 }
 
+// TODO (grzkv): The name does not say that it tries to re-connect
```

Comment:
Indeed, with the last change, the name does not convey the current logic, and needs to be updated.